minigalaxy/launcher.py

```diff
@@ -79,6 +79,14 @@ def __get_execute_command(game) -> list:
         prefix = os.path.join(game.install_dir, "prefix")
         os.environ["WINEPREFIX"] = prefix
 
+        # Enable ACO by default for AMD users.
+        # Mesa 19.x and > are needed.
+        # ACO allow to compile more quickly than with LLVM
+        # and avoid stuttering in Vulkan games (only).
+        # This option is not enabled by default
+        # We'll can remove this variable when it'll be done
+        os.environ["RADV_PERFTEST"] = "aco"
```

Comment:
Is this needed for this?

Reply:
No, it's not needed to display FPS Vulkan but i did not want to do an other PR only for that ... :X
It's an other option no related to FPS display

Reply:
Ah okay. Could you remove it from this PR then?